Exclude test in find_packages

strigazi opened the pull request:
To avoid conflicts with projects that have tests in their top-level
directory:

* exlude tests in find_packages
* add include in MANIFEST.in

Closes: #688

Signed-off-by: Spyros Trigazis <spyridon.trigazis@cern.ch>

**Note**: oauth2client is now deprecated. As such, it is unlikely that we will
review or merge to your pull request. We recommend you use
[google-auth](https://google-auth.readthedocs.io) and [oauthlib](http://oauthlib.readthedocs.io/).

strigazi commented:
I'll sign the CLA.

Please review

strigazi commented:
I signed it!

strigazi commented:
A point release would be nice after this goes in!

theacodes commented on MANIFEST.in:
```diff
@@ -1,3 +1,4 @@
 include README.md
```
Can you change this whole file to:

```
include README.md LICENSE CHANGELOG.md
recursive-include tests *
```

strigazi replied:
DONE

theacodes commented on setup.py:
```diff
@@ -61,7 +61,7 @@
     url='http://github.com/google/oauth2client/',
     install_requires=install_requires,
-    packages=find_packages(),
+    packages=find_packages(exclude=("tests",)),
```
Please use `tests*` to avoid potentially discovering sub-packages of `tests`.

Also, this is a nit, but can you use single quotes here like the rest of the module?

strigazi replied:
DONE

strigazi commented:
Does the googlebot take that long to be updated?

jonparrott commented:
@strigazi it's pretty instant. Are you sure you signed the CLA and authored your commits with an email associated with your account?

jonparrott commented:
@strigazi github is showing an unknown author for [this commit](https://github.com/google/oauth2client/pull/720/commits/aa4024261ec34c09a84322d54d91649b95cb8fb9).

strigazi commented:
I can try my gmail. I have my CERN email as a secondary email in google account.

jonparrott commented:
@strigazi this issue is the email on your commits which isn't associated with your github account. If you add your cern email to your github account, it should all work. :)

strigazi committed:
Exclude test in find_packages

To avoid conflicts with projects that have tests in their top-level
directory:

* exlude tests in find_packages
* add include in MANIFEST.in

Closes: #688

Signed-off-by: Spyros Trigazis <strigazi@gmail.com>

jonparrott commented:
Thanks, @strigazi! I will merge when CI passes.

strigazi commented:
So, it works only with my gmail :(

strigazi commented:
@jonparrott Can you also do a point release after?

jonparrott commented:
@strigazi if you'll make a release PR ([example](https://github.com/google/oauth2client/pull/718))

jonparrott commented:
(Travis passed, system test failures are expected on external PRs)
